src/ui/Quickview/Quickview.ts

```diff
@@ -259,6 +254,53 @@ export class Quickview extends Component {
     }
   }
 
+  private buildContent() {
```

Comment:
I feel this method is doing a little too much. What do you think of extracting everything related to the tooltip into a method `buildTooltip`? e.g.
```
const icon = this.buildIcon();
const tooltip = this.buildTooltip(icon); // this would return the "caption"

const content = $$('div');
content.append(icon);
content.append(tooltip);

$$(this.element).append(content.el);
```